docs/admin_guide.md

```diff
@@ -908,7 +908,7 @@ With the following content:
 
 ```ini
 [program:messenger]
-command=php /var/www/mbin/bin/console messenger:consume async async_ap failed --time-limit=3600
+command=php /var/www/mbin/bin/console messenger:consume scheduler_default async async_ap failed --time-limit=3600
```

Comment:
Doesn't `config/packages/messenger.yaml` need to be updated to include this new `scheduler_default` queue?

Reply:
nope. It has actually nothing to do with the messengers besides the consume command: https://symfony.com/doc/current/scheduler.html#symfony-scheduler-basics